doc/source/tutorial/stats/probability_distributions.rst

```diff
@@ -602,11 +602,11 @@ the probabilities.
    :include-source: 0
 
 
-Next, we can test whether our sample was generated by our norm-discrete
-distribution. This also verifies whether the random numbers were generated
-correctly.
+Next, we can use a chi-squared test, `stats.chisquare`, to test the null
```

Comment:
Just wanted to check whether this is intended as a clickable link?
```suggestion
Next, we can use a chi-squared test, :func:`scipy.stats.chisquare`, to test the null
```

Reply:
Yes, thanks!